### PR TITLE
docs/managed_vms/endpoints: add /auth/info/firebase

### PR DESCRIPTION
**Pull request by broady**

### CODE REVIEW

**Comment by broady** [comment]
@jonwayne PTAL

**Comment by broady** [comment]
@jonparrott PTAL

File: docs/managed_vms/endpoints/app.go [hunk 1]
```diff
@@ -51,8 +53,22 @@ func echoHandler(w http.ResponseWriter, r *http.Request) {
...
+			w.Header().Set("Access-Control-Allow-Headers", "Authorization")
+			return
+		}
+		h(w, r)
+	}
+}
+
 // authInfoHandler reads authentication info provided by the Endpoints proxy.
 func authInfoHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Access-Control-Allow-Origin", "*")
```
Comment (jonparrott) [inline docs/managed_vms/endpoints/app.go]: couldn't the `corsHandler` set this always?

**Comment by jonparrott** [comment]
LGTM

File: docs/managed_vms/endpoints/app.go (hunk 1 above)
Comment (broady) [inline docs/managed_vms/endpoints/app.go]: And it does. I forgot to revert this line. We don't want it for the other handlers.

**Pull request merged**